pypdf/_writer.py

```diff
@@ -1349,6 +1371,97 @@ def add_metadata(self, infos: Dict[str, Any]) -> None:
         assert isinstance(self._info, DictionaryObject)
         self._info.update(args)
 
+    def compress_identical_objects(
+        self,
+        remove_identicals: bool = True,
+        remove_orphans: bool = True,
+        verbose: int = -1,
+    ) -> None:
+        """
+        Parse the Pdf file and merge objects that have same harsh.
+        This will make objects common to multiple pages
+        Recommended to be used just before writing output
+
```

Comment:
Pdf -> PDF
harsh -> hash

And please add the trailing periods to the sentences.